server/Dockerfile.dev

```diff
@@ -1,4 +1,6 @@
-FROM oven/bun:latest
+FROM oven/bun:alpine
+
+RUN apk --no-cache add curl
```

Comment:
following previous, since the health check on dev server itself may not be necessary, we won't need `curl`

Reply:
why not have health check on dev server. also i was using it as a check before starting nginx service

Reply:
because if we don't need nginx reverse proxy, then we don't need healthcheck
it may be useful if we need nginx reverse proxy
but the purpose of this docker-compose at the moment is solely for backend development, that's why we don't need reverse proxy in order to keep it more light-weighted

not sure if we're going with adding nginx for frontend in the same repo, but if that's the case we don't need nginx reverse proxy either because our frontend will be a standalone server, and we can simply make that another service

if it's a prod docker-compose we can use nginx reverse proxy for setting up https, serving static assets and so on